### ISSUE TITLE
Recipient list is empty  Android_Intent_Plus  'android.intent.extra.EMAIL'

### ISSUE THREAD

**Issue by jk2pr** [fluttercommunity/plus_plugins#620]
I tried to use this plugin to send email 

 ```
final AndroidIntent androidIntent = AndroidIntent(
      action: 'action_view',
      arguments: <String, dynamic>{
        'android.intent.extra.EMAIL': <String>[test@test.com],
        'android.intent.extra.SUBJECT': 'SUBJECT',
        'android.intent.extra.TEXT': HELLO,
      },
      data: 'mailto:',
    );
 if (await androidIntent.canResolveActivity()) {
      await androidIntent.launchChooser('Choose');
```

but I can see empty list in recipient box of email client (outlook and gmail)

**Comment by harriseldon** [comment]
I did some testing on this on a way to get it to work. It may not be optimal for you, but the data is expected to be a Uri. I was able to get it to work by passing the emails in the data link as follows:
For a single:
``` dart

  data: Uri.encodeFull('mailto:test@test.com'),
```
For multiple:
``` dart
data: Uri.encodeFull(
          'mailto:newperson@myaddress.com,otherperson@place.com,John Smith <thejohn@thejohn.com>'),
```

I have gone too far into the detail of the how the intent is passed to the android layer, but at least this is a working solution.

*Note: I also tried with the action **'android.intent.action.SENDTO'**. It works for both the launch method and the launchChooser method. I found that the launch method when using the SENDTO action automatically opened my default email application, while using action_view prompted me to choose a default or one time.*
